CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ Breaking changes:
 `Timber.tag` is no longer supported by our [Timber integration](https://docs.sentry.io/platforms/android/configuration/integrations/timber/) and will not appear on Sentry for error events. 
 Please vote on this [issue](https://github.com/getsentry/sentry-java/issues/1900), if you'd like us to provide support for that.
 
+* Fix: NPE while adding "response_body_size" breadcrumb, when response body length is unknown (#1907)
```

Comment:
Please move this line to line 6, above breaking changes.

Reply:
@maciejwalkowiak All set